datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/Digest.kt

```diff
@@ -14,4 +15,10 @@ enum class Digest(override val oid: ObjectIdentifier) : Identifiable {
     SHA384(KnownOIDs.sha_384),
     SHA512(KnownOIDs.sha_512);
 
-}
\ No newline at end of file
+    val outputLength: BitLength inline get() = when(this) {
+        SHA1 -> 160u
+        SHA256 -> 256u
+        SHA384 -> 384u
+        SHA512 -> 512u
+    }.let(::BitLength)
```

Comment:
going to need this for the provider

Reply:
add this as constructor parameter, please